geo/geo_computation.py

```diff
@@ -19,7 +21,14 @@
 
 def main(argv):
   del argv  # Unused.
-  print(compute_route((FLAGS.orig_lat,FLAGS.orig_lon), (FLAGS.dest_lat,FLAGS.dest_lon)))
+
+  # compute graph over Manhattan
+  graph = ox.graph_from_place('Manhattan, New York City, New York, USA')
+
+  #Convert a graph to nodes and edge GeoDataFrames.
+  nodes, _ = ox.graph_to_gdfs(graph)
+
+  print(compute_route(Point(FLAGS.orig_lat,FLAGS.orig_lon), Point(FLAGS.dest_lat,FLAGS.dest_lon),graph,nodes))
```

Comment:
Break this up so it doesn't go past 80 characters on a single line. Also, make sure to have spaces after commas.